vault/identity_store_oidc_test.go

```diff
@@ -1432,9 +1433,10 @@ func TestOIDC_GetKeysCacheControlHeader(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	// headerVal will be a random value between 0 and jwksCacheControlMaxAge
-	if headerVal > int(jwksCacheControlMaxAge) {
-		t.Fatalf("unexpected header value, got %d", headerVal)
+	// headerVal will be a random value between 0 and jwksCacheControlMaxAge (in seconds)
+	if headerVal > durationSeconds {
+		t.Logf("jwksCacheControlMaxAge: %d", int(jwksCacheControlMaxAge))
```

Comment:
Nit: Since you're specifying the output format to be `%d`, you don't have to do the conversion here.
```suggestion
		t.Logf("jwksCacheControlMaxAge: %d", jwksCacheControlMaxAge)
```